apps/bors_github/lib/github/server.ex

```diff
@@ -390,6 +390,31 @@ defmodule BorsNG.GitHub.Server do
   end
 
   @doc """
+  Uses a token from the cache, or, if the request fails,
+  retry without using the cached token.
+  """
+  @spec use_token!(ttoken, ttokenreg, ((ttoken) -> term)) ::
+    {:reply, term, ttokenreg}
+  def use_token!({:installation, installation_xref} = token, state, fun) do
```

Comment:
I see the method is defined with the `!` at the end so I traced the calls to find what raises the exception.
It's `HTTPoison.post!` right?